.github/workflows/tests.yml

```diff
@@ -0,0 +1,22 @@
+name: Tests
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
There's more to this integration. For example, when I open a pull request, it should run the tests on the pull request. I'll look into this a little later.